sdk/src/asset_handlers/bmff_io.rs

```diff
@@ -1684,16 +1692,22 @@ impl CAIWriter for BmffIO {
         // if the incoming Store has an update manifest we must split it into original and update stores
         let mut validation_log =
             StatusTracker::with_error_behavior(ErrorBehavior::StopOnFirstError);
-        let (pc, is_update) = if let Ok(store) = Store::from_jumbf(store_bytes, &mut validation_log)
-        {
-            let pc = store
-                .provenance_claim()
-                .ok_or(Error::BadParam("no provenance claim".to_string()))?;
-            let is_update = pc.update_manifest();
-            (Some(pc.clone()), is_update)
-        } else {
-            (None, false)
-        };
+
+        let settings = Settings::default();
+        // ^^ TO REVIEW: Is there a reason we'd need non-default settings here?
+        // If so, that means we hae to rework CAIReader and CAIWriter traits
+        // to take a &Settings arg and that is a pretty huge change.
```

Comment:
I think `Settings` in the `Store` constructors are only used for configuring trust. In this scenario we are only using the `Store` to reconstruct the split manifest stores in BMFF, the trust is validated with a higher-level `Store`. It does feel wrong to create default settings and pass it in as a parameter here, but I don't think it'll be a problem for the time being.

cc @mauricefisher64 to confirm.